FinalChallenge/HouseRepresentation/ViewModels/BluePrintViewControllerModel.swift

```
@@ -48,7 +48,7 @@ class BluePrintViewControllerModel<HouseRepository: RepositoryErasurer<Isometric
         
         self.houseRepository.saveEntity?(newIsometricHouse ?? self.isometricHouse) { result in
             switch result {
-            case .error(let error):
+            case .error:
```

Perhaps it should be better to throw an error instead of break this case of the enum

---

não é meu código =/